components/Footer.tsx

```diff
@@ -9,7 +9,9 @@ const Footer: React.FC<Props> = ({ footerType }) => {
     ? `fw-light text-center ${footerType}`
     : `mt-5 fw-light text-center pb-5 text-muted`
   return (
-    <footer className={footerClass}>&copy; Copyright 2022 GarageScript</footer>
+    <footer className={footerClass}>
+      &copy; Copyright {String(new Date()).split(' ')[3]} GarageScript
```

Comment:
I think `new Date().getFullYear()` would be more clear.

Reply:
Making the change now